README.md

```diff
@@ -45,7 +45,11 @@ Then, import `SwiftSyntax` in your Swift code.
 
 ## Documentation
 
-Documentation can be found [here](Documentation) and some examples of using SwiftSyntax can be found [here](Examples).
+Documentation can be found in the following places:
+- For the new parser: [Sources/SwiftParser/SwiftParser.docc](Sources/SwiftParser/SwiftParser.docc) 
```

Comment:
This word "new" will be outdated (hopefully) soon. Can we just call it "the parser"?

Same for "the new Swift parser" in Documentation/SwiftParser.md